src/permissions/__tests__/fallbackError.test.js

```diff
@@ -32,7 +32,7 @@ const contextValue = {
 }
 
 describe('fallbackError', () => {
-  test('throws an error with the requestId present', async () => {
+  test('throws an error with the requestId present (cmr-error)', async () => {
```

Comment:
```suggestion
  describe('when the thrown error is a CMR error', () => {
    test('throws an error with the requestId present', async () => {
```

Organizing tests this way does a better job of telling the story of the test. 

describe: when given this input
test: this is the expected output